modules/_labs/color-picker/react/lib/parts/SwatchBook.tsx

```diff
@@ -4,6 +4,8 @@ import {borderRadius, colors, spacing} from '@workday/canvas-kit-react-core';
 import {focusRing, mouseFocusBehavior} from '@workday/canvas-kit-react-common';
 import {ColorSwatch} from '@workday/canvas-kit-react-color-picker';
 
+const numberOfSwatchColumns: number = 8;
```

Comment:
```suggestion
const numberOfSwatchColumns = 8;
```

Adding the `number` widens the type to `number` instead of `8`